### PR TITLE
Merge staging production

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h3>Confirmation</h3>
 
 	<div class="modal-body clearfix" id="export-body">
 		<form name="newRole" novalidate class="" autocomplete="off">
-			<h3>Are you sure want to delete <b *ngIf="selectedRoleForDelete">{{selectedRoleForDelete.name}}</b>?</h3>
+			<h3>Are you sure you want to delete <b *ngIf="selectedRoleForDelete">{{selectedRoleForDelete.name}}</b>?</h3>
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/permissions/components/confirmation/confirmation.model.component.html]: Need to review this

File: electron-sign/notarize.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    try {
+        await electronNotarize.notarize({
+            appBundleId: appId,
+            appPath: appPath,
+            appleId: 'rohit@walkover.in', // enter apple id here
+            appleIdPassword: 'Mac@1234'
+        });
+    } catch (error) {
+        console.error(error);
```
Comment (ravinderwalkover) [inline electron-sign/notarize.js]: Remove this console

File: apps/electron-giddh/src/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "giddh",
-    "version": "7.0.0",
+    "version": "6.9.9",
```
Comment (rohit-walkover) [inline apps/electron-giddh/src/package.json]: Update version 7.0.1

File: apps/web-giddh/src/app/app.component.ts [hunk 4]
```diff
@@ -26,19 +27,24 @@ import { reassignNavigationalArray } from './models/defaultMenus'
         './app.component.css'
     ],
     template: `
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/app.component.ts]: Why this template params there need to user .html file for this.

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 5]
```diff
@@ -727,6 +731,13 @@ ul.sub-list {
     }
     .nav-left-bar .company-name-finance-year {
         width: 200px;
+        float: left;
+    }
+    .cText[_ngcontent-ytw-c2] {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.scss]: remove .cText params and don't use angular custom CSS

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 6]
```diff
@@ -727,6 +731,13 @@ ul.sub-list {
...
+        overflow: hidden;
+    }
+    .cText.head-nav {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.scss]: Don't use display none use *ngIf

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts [hunk 7]
```diff
@@ -364,22 +368,29 @@ export class InvoicePreviewComponent implements OnInit, OnChanges, OnDestroy {
...
+            this.store.pipe(select(store => store.receipt.vouchers), publishReplay(1), refCount()),
+            this.store.pipe(select(store => store.purchaseRecord.updatedRecordDetails))
+        ]).pipe(takeUntil(this.destroyed$))
+        .subscribe((response) => {
+            const voucherData: ReciptResponse = response[0];
+            const record: PurchaseRecordUpdateModel = response[1];
+            if (voucherData && record) {
                 this.selectedInvoiceForDetails = null;
-                let voucherIndex = this.voucherData.items.findIndex(item => item.uniqueName === record.purchaseRecordUniqueName);
+                let voucherIndex = voucherData.items.findIndex(item => item.uniqueName === record.purchaseRecordUniqueName);
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts]: check nullity for voucherData

File: apps/web-giddh/src/app/invoice/preview/models/bulkUpdateModal/invoiceBulkUpdateModal.component.ts [hunk 8]
```diff
@@ -11,26 +11,41 @@ import { ToasterService } from '../../../../services/toaster.service';
...
     public fieldOptions: IOption[] = [
-        { label: 'PDF Template', value: 'pdf-template' },
+        { label: 'PDF Template', value: 'pdfTemplate' },
         { label: 'Notes', value: 'notes' },
         { label: 'Signature', value: 'signature' },
-        { label: 'Due Date', value: 'due-date' },
-        { label: 'Shipping Details', value: 'shipping-details' },
-        { label: 'Custom Fields', value: 'custom-fields' }
+        { label: 'Due Date', value: 'dueDate' },
+        { label: 'Shipping Address', value: 'shippingDetails' },
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/invoice/preview/models/bulkUpdateModal/invoiceBulkUpdateModal.component.ts]: Remove shipping details and bulk update

File: apps/web-giddh/src/app/settings/settings.component.ts [hunk 9]
```diff
@@ -192,24 +196,53 @@ export class SettingsComponent implements OnInit {
...
+                "refresh_token": p.refresh_token
+            };
+            this._authenticationService.saveGmailToken(dataToSave).subscribe((res) => {
+
+                if (res.status === 'success') {
+                    this._toast.successToast('Gmail account added successfully.', 'Success');
+                } else {
+                    this._toast.errorToast(res.message, res.code);
+                }
+                this.store.dispatch(this.settingsIntegrationActions.GetGmailIntegrationStatus());
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/settings/settings.component.ts]: method name should be camel casing.

**Pull request merged**